lte/gateway/fabfile.py

```diff
@@ -547,7 +547,7 @@ def _start_gateway_containerized():
         run_with_retry('DOCKER_REGISTRY=%s docker-compose -f docker-compose.yaml up -d --quiet-pull' % (env.DOCKER_REGISTRY))
 
 
-def run_with_retry(command, retries=3):
+def run_with_retry(command, retries=10):
```

Comment:
Should there be some kind of sleep here between retries?

Reply:
[No.](https://github.com/magma/magma/pull/14090#issuecomment-1274457628)

Reply:
What's the time the services take to restart, and what's the time a retry usually takes? If we don't know that they're on the same order of magnitude, increasing the number of retries by a small factor might not change anything.

We can try increasing the count anyway and maybe the `docker ps` will help debugging, but I'm not sure this will solve the issue.

Reply:
I definitely think there should be a small sleep during retries. Just relying on the check itself to be slow seems strange to me, and also wastes resources.

Reply:
The `run_with_retry` from the PR that you link has the same issue.

Reply:
3s sleep added.